plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerAbandonedResourcesCleaner.java

```diff
@@ -89,6 +93,10 @@ public DockerAbandonedResourcesCleaner(
   )
   @Override
   public void run() {
+    if (skipCleaningProperty != null && Boolean.parseBoolean(skipCleaningProperty)) {
+      LOG.info("Skipping che unused containers and network cleanup ...");
```

Comment:
FYI this code prevents the execution of code of cleanups but doesn't prevent spawning jobs periodically.